packages/arb-token-bridge-ui/src/components/TransferPanel/TokenSearch.tsx

```diff
@@ -240,6 +242,25 @@ function TokenRow({ style, onClick, token }: TokenRowProps): JSX.Element {
     )
   }, [isDepositMode, l1Network, l2Network])
 
+  const tokenBalanceContent = useMemo(() => {
+    return tokenIsAddedToTheBridge ? (
+      <span className="flex items-center whitespace-nowrap text-sm text-gray-500">
+        {tokenBalance ? (
+          formatAmount(tokenBalance, {
+            decimals: token?.decimals,
+            symbol: tokenSymbol
+          })
+        ) : (
+          <div className="mr-2">
+            <Loader color="#28A0F0" size="small" />
+          </div>
+        )}
+      </span>
+    ) : (
+      <span className="text-sm font-medium text-blue-link">Import</span>
+    )
```

Comment:
since we're in a `useMemo`, we should try to eliminate ternary and use `if`s with `return`s whenever possible

Reply:
updated